src/Service/Indexer/SiteKit/DefaultSchema2xDocumentEnricher.php

```diff
@@ -0,0 +1,457 @@
+            $doc->exclude_groups = array_map(
+                fn($id): string => (string)$this->idWithoutSignature($id),
+                $groups
+            );
+        } else {
+            $doc->exclude_groups = ['none'];
+            $doc->include_groups = ['all'];
+        }
+
+        $doc->sp_source = ['internal'];
+
+        return $this->enrichContent($resource, $doc);
+    }
+
+    /**
+     * @param IndexSchema2xDocument $doc
+     * @return IndexSchema2xDocument
+     */
+    private function enrichContent(
+        Resource $resource,
+        IndexDocument $doc,
+    ): IndexDocument {
+
+        $content = [];
+        $content[] = $resource->getData()->getString(
+            'searchindexdata.content'
+        );
+
+        $content[] = $this->contentCollector->collect(
+            $resource->getData()->getArray('content')
+        );
+
+        /** @var ContactPoint $contactPoint */
+        $contactPoint = $resource->getData()->getArray('metadata.contactPoint');
+        $content[] = $this->contactPointToContent($contactPoint);
+
+        /** @var array<array{name?:string}> $categories */
+        $categories = $resource->getData()->getArray('metadata.categories');
+        foreach ($categories as $category) {
+            $content[] = $category['name'] ?? '';
+        }
+
+        $cleanContent = preg_replace(
+            '/\s+/',
+            ' ',
+            implode(' ', $content)
+        );
+
+        $doc->content = trim($cleanContent ?? '');
+
+        return $doc;
+    }
+
+    /**
+    * @param ContactPoint $contactPoint
+    * @return string
+     */
+    private function contactPointToContent(array $contactPoint): string
+    {
+        if (empty($contactPoint)) {
+            return '';
+        }
+
+        $content = [];
+        foreach (($contactPoint['contactData']['phoneList'] ?? []) as $phone) {
+            $countryCode = $phone['phone']['countryCode'] ?? '';
+            if (
+                !empty($countryCode) &&
+                !in_array($countryCode, $content, true)
+            ) {
+                $content[] = '+' . $countryCode;
+            }
+            $areaCode = $phone['phone']['areaCode'] ?? '';
+            if (!empty($areaCode) && !in_array($areaCode, $content, true)) {
+                $content[] = $areaCode;
+                $content[] = '0' . $areaCode;
+            }
+            $content[] = ($phone['phone']['localNumber'] ?? '');
+        }
+        foreach (($contactPoint['contactData']['emailList'] ?? []) as $email) {
+            $content[] = $email['email'];
+        }
+
+        if (isset($contactPoint['addressData'])) {
+            $addressData = $contactPoint['addressData'];
+            $content[] = ($addressData['street'] ?? '');
+            $content[] = ($addressData['buildingName'] ?? '');
+            $content[] = (
+                $addressData['postOfficeBoxData']['buildingName'] ?? ''
+            );
+        }
```

Comment:
all those brackets are superfluous

```suggestion
            $content[] = $phone['phone']['localNumber'] ?? '';
        }
        foreach ($contactPoint['contactData']['emailList'] ?? [] as $email) {
            $content[] = $email['email'];
        }

        if (isset($contactPoint['addressData'])) {
            $addressData = $contactPoint['addressData'];
            $content[] = $addressData['street'] ?? '';
            $content[] = $addressData['buildingName'] ?? '';
            $content[] = $addressData['postOfficeBoxData']['buildingName']
                ?? '';
        }
```